types/draft.ts

```diff
@@ -0,0 +1,32 @@
+import { z } from 'zod'
+const Draft = z.object({
```

Comment:
The updated schema is publicly deployed now at https://tulsawebdevs.org/docs/api/

We've got some min/max values here that need updating.

Reply:
The range for title in the schema is <= 48. Does that mean the min value should be set to 0?